### PR TITLE
Update Youtube Publication documentation

### PR DESCRIPTION
**Pull request by Arnei**
The documentation for publishing to youtube is a bit outdated. I tried updating it based on my experiences with setting it up.

### CODE REVIEW

File: docs/guides/admin/docs/modules/youtubepublication.md [hunk 1]
```diff
@@ -75,34 +81,16 @@ With the JSON file created and saved previously, you have to proceed as describe
...
-
-        <input id="publishToYouTube" name="publishToYouTube" type="checkbox" class="configField" value="true"/>
-
-- Open the workflows `etc/opencast/workflows/ng-retract.xml`
-
-- In the file, modify the `<configuration_panel>` and enable the YouTube option, like this:
+## Add YouTube publication to your Opencast workflows
 
-        <input id="retractFromYouTube" type="checkbox" class="configField" value="true" disabled="disabled" />
+Opencast can now publish to YouTube. To make use of this, add the [Publish Youtube workflow operation](../workflowoperationhandlers/publish-youtube-woh.md)
```
Comment (JulianKniephoff) [inline docs/guides/admin/docs/modules/youtubepublication.md]:
```suggestion
Opencast can now publish to YouTube. To make use of this, add the [Publish YouTube workflow operation](../workflowoperationhandlers/publish-youtube-woh.md)
```

File: docs/guides/admin/docs/modules/youtubepublication.md [hunk 2]
```diff
@@ -75,34 +81,16 @@ With the JSON file created and saved previously, you have to proceed as describe
...
+to your Opencast workflows. You can find more details on how to use the operation on its [own page]((../workflowoperationhandlers/publish-youtube-woh.md)).
+In general, it should be placed near your other `publish` operations.
 
-  becomes
+You may also want to add the [Retract Youtube workflow operation](../workflowoperationhandlers/retract-youtube-woh.md)
```
Comment (JulianKniephoff) [inline docs/guides/admin/docs/modules/youtubepublication.md]:
```suggestion
You may also want to add the [Retract YouTube workflow operation](../workflowoperationhandlers/retract-youtube-woh.md) 
```

File: docs/guides/admin/docs/modules/youtubepublication.md [hunk 3]
```diff
@@ -75,34 +81,16 @@ With the JSON file created and saved previously, you have to proceed as describe
...
-
-- Open the workflows `etc/opencast/workflows/ng-retract.xml`
-
-- In the file, modify the `<configuration_panel>` and enable the YouTube option, like this:
+## Add YouTube publication to your Opencast workflows
 
-        <input id="retractFromYouTube" type="checkbox" class="configField" value="true" disabled="disabled" />
+Opencast can now publish to YouTube. To make use of this, add the [Publish Youtube workflow operation](../workflowoperationhandlers/publish-youtube-woh.md)
+to your Opencast workflows. You can find more details on how to use the operation on its [own page]((../workflowoperationhandlers/publish-youtube-woh.md)).
+In general, it should be placed near your other `publish` operations.
```
Comment (JulianKniephoff) [inline docs/guides/admin/docs/modules/youtubepublication.md:89]: I'm not sure what you mean by this. Don't all workflow definitions have to live in `etc/workflows`? 🤔

File: docs/guides/admin/docs/modules/youtubepublication.md [hunk 4]
```diff
@@ -75,34 +81,16 @@ With the JSON file created and saved previously, you have to proceed as describe
...
 
-  becomes
+You may also want to add the [Retract Youtube workflow operation](../workflowoperationhandlers/retract-youtube-woh.md) 
+near your `retract` operations.
```
Comment (JulianKniephoff) [inline docs/guides/admin/docs/modules/youtubepublication.md:92]: See above.

**Review by mliradelc — APPROVED** [review]
I asked another user to try this new configuration, first time he tried to configure Opencast with YouTube. He answered me that the new documentation works.

I validate this documentation.

**Review by JulianKniephoff — CHANGES_REQUESTED** [review]
I haven't tested it yet, but here are some nitpicks already. 😅

File: docs/guides/admin/docs/modules/youtubepublication.md (hunk 3 above)
Comment (mliradelc) [reply docs/guides/admin/docs/modules/youtubepublication.md:89]: Indeed, but he is not talking about the `file` location but rather from inside the workflow. Place the step to publish next to the other publish steps in your workflow.

Comment (JulianKniephoff) [reply docs/guides/admin/docs/modules/youtubepublication.md:89]: Oh yeah, my bad! 🙈

File: docs/guides/admin/docs/modules/youtubepublication.md [hunk 5]
```diff
@@ -18,22 +18,28 @@ pages and we don't always notice!
...
-- Login to Google account
+- Login to your Google account
 - Navigate to the [**Google Developers Console**][googledevconsole]
 - Click **Create Project** and follow the instructions
 - Navigate to the [**Google Credentials Console**][googleapiconsole]
 - Select **OAuth consent screen**
-- Configure the API Consent Screen, you will need to set the Product name
+- Configure the API Consent Screen
+  - For **User type** select `external` 
+  - Fill in the rest of the required settings at discretion
```
Comment (JulianKniephoff) [inline docs/guides/admin/docs/modules/youtubepublication.md]: These do not show up as indented in the rendered docs which confused me just now. 🤔

File: docs/guides/admin/docs/modules/youtubepublication.md [hunk 6]
```diff
@@ -18,22 +18,28 @@ pages and we don't always notice!
...
 - Select **Credentials**
 - Select **Create Credentials**, specifically OAuth Client ID
-- Select **Other** application type
+  - **Application type** must be set to `Desktop`
```
Comment (JulianKniephoff) [inline docs/guides/admin/docs/modules/youtubepublication.md]: Same here.

**Review by JulianKniephoff — CHANGES_REQUESTED** [review]
Found some minor issues with the docs while trying to follow them, still.

In addition to the inline comments, I ran into the trap that you need to add a test user in the "Configure the API Consent Screen" step when your app is still in testing mode. Otherwise you get an "access denied" in the verification step. Maybe we should hint at this. 🤷‍♀️ 

I also think that the two restarts in the guide aren't actually necessary, so maybe get rid of those as well?

File: docs/guides/admin/docs/modules/youtubepublication.md (hunk 5 above)
Comment (ziegenberg) [reply docs/guides/admin/docs/modules/youtubepublication.md]: This probably needs four spaces of indentation. See line 38 in this file

Comment (Arnei) [reply docs/guides/admin/docs/modules/youtubepublication.md]: Thanks for catching this.

**Comment by Arnei** [comment]
> I also think that the two restarts in the guide aren't actually necessary, so maybe get rid of those as well?

Did you successfully set this up without restarting?

**Comment by JulianKniephoff** [comment]
Yes I did. The URL that is mentioned just appears in the log and you can continue to use Opencast even before going through that.

However, it **is** true that if you (re-)start Opencast without having completed that step, stuff won't start up correctly. 🤔

**Review by JulianKniephoff — CHANGES_REQUESTED** [review]
I think ~~the last commit~~ 3b63b7aa127b670c265435f0f563b2d1a704fb9d includes some accidental config changes.

**Pull request merged**